library/std/src/backtrace.rs

```diff
@@ -200,6 +197,7 @@ impl fmt::Debug for Backtrace {
     }
 }
 
+#[unstable(feature = "backtrace_frames", issue = "79676")]
```

Comment:
Trait impls are always insta-stable.

Reply:
The BacktraceFrame type is unstable and requires the backtrace_frames feature, so wouldn't an impl for it need to be the same? It definitely wouldn't work to say backtrace_frames is stable here while saying it's unstable elsewhere.

Reply:
I missed that this was an impl on an unstable type. Unstable impls on stable types aren't supported (rustc doesn't actually check stability for impls), but unstable impls for unstable types are just fine.